apps/dashboard/lib/trpc/routers/key/create.ts

```diff
@@ -17,21 +21,14 @@ export const createKey = t.procedure
   .use(requireWorkspace)
   .input(createKeyInputSchema)
   .mutation(async ({ input, ctx }) => {
-    const keyAuth = await db.query.keyAuth
-      .findFirst({
-        where: (table, { and, eq }) =>
-          and(eq(table.workspaceId, ctx.workspace.id), eq(table.id, input.keyAuthId)),
-        with: {
-          api: true,
-        },
-      })
-      .catch((_err) => {
```

Comment:
why are we removing the catches in the mutations? now we're passing the raw sql errors to the frontend again